training/src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,106 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
When you use Postgresql, you should use identity instead of auto strategy. This is because the default strategy use a global incremental that is shared between all tables.
```suggestion
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```